cmd/web3/main.go

```diff
@@ -1320,12 +1328,22 @@ func GetAddressDetails(ctx context.Context, network web3.Network, addrHash, priv
 		return
 	}
 
+	var blockN *big.Int
+	var err error
+	// Don't try to parse empty string, which means 'latest'.
+	if blockNumber != "" {
+		blockN, err = web3.ParseBigInt(blockNumber)
+		if err != nil {
+			fatalExit(fmt.Errorf("Block argument must be a number (decimal integer) or hash (hexadecimal with 0x prefix) %q: %v", blockNumber, err))
```

Comment:
Is this meant to support hex hash as well? Looks like it only parses int

Reply:
updated (supports only Int)